### PR TITLE
Sort YAML alphabetically

### PR DESCRIPTION
**Pull request by amire80**
Also add an automatic test for that.

### CODE REVIEW

**Comment by amire80** [comment]
I think it's correct now. Can be reviewed.

File: composer.json [hunk 1]
```diff
@@ -50,6 +50,7 @@
 	"scripts": {
 		"fix": "phpcbf",
 		"cs": "phpcs -sp",
+		"yaml-sort-check": "composer require mhujer/yaml-sort-checker; yaml-sort-checker",
```
Comment (Abijeet) [inline composer.json:53]: `composer require mhujer/yaml-sort-checker` library can be added to this section: https://github.com/wikimedia/language-data/blob/master/composer.json#L36

**Review by Abijeet — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (amire80) [reply composer.json:53]: That's what I did initially. The reason I switched to this hack is that the library I use for testing YAML sorting is incompatible with PHP < 8.1, and require in composer.json applies to all PHP versions.

Comment (Abijeet) [reply composer.json:53]: OK. Can you add a comment in `.github/workflows/php.yml` regarding that?

Comment (amire80) [reply composer.json:53]: Done.

**Review by Abijeet — APPROVED** [review]

**Pull request merged**